krank.cabal

```diff
@@ -46,7 +46,8 @@ test-suite krank-test
                        , hspec >= 2.7 && < 2.8
                        , PyF >= 0.8.1.0 && < 0.9
                        , krank
-                       , megaparsec
+                       , pcre-heavy
+                       , bytestring
```

Comment:
nit: do we want to keep alphabetical ordering for easier management?

Reply:
I had never really cared about sorted items in list as long as there is no automatic tool to ensure the policy,

I'll fix that. Thank you

Reply:
Not fixed for now. I'd like to have something automatic to check for these, else I don't think there is value.